src/QMCWaveFunctions/tests/test_multi_slater_determinant.cpp

```diff
@@ -179,6 +179,16 @@ void test_LiH_msd(const std::string& spo_xml_string,
 
     CHECK(std::real(ratios2[0]) == Approx(-0.8544310407));
     CHECK(std::real(ratios2[1]) == Approx(-1.0830708458));
+
+    std::fill(ratios2.begin(), ratios2.end(), 0);
+    Matrix<ValueType> dratio(2, nparam);
+    twf.evaluateDerivRatios(VP, active, ratios2, dratio);
+
+    CHECK(std::real(ratios2[0]) == Approx(-0.8544310407));
```

Comment:
Where did these values come from?

Reply:
ratios2 should be identical to the return of evaluateRatios call above.
dratio value is obtained from the old code path by accepting a move and then calling evaluateDeriv.